features/tags_and_wrangling/tag_wrangling_more.feature

```diff
@@ -141,10 +141,83 @@ Feature: Tag wrangling: assigning wranglers, using the filters on the Wranglers
+       And I go to the tagwranglings page
+       And I follow "Fandoms by media (2)"
+       And I check the wrangling option for "Cowboy Bebop"
+       And I press "Wrangle"
+     Then I should not see "Cowboy Bebop"
+
+  Scenario: more wrangling....Relationships
+    Given the following typed tags exists
+        | name                                   | type         | canonical |
+        | Tybalt (Toby Daye)                     | Character    | false     |
+        | October Daye                           | Character    | false     |
+        | Toby Daye/Tybalt                       | Relationship | true      |
+        | October Daye Series - Seanan McGuire   | Fandom       | false     |
+      And a media exists with name: "Books & Literature", canonical: true
+      And I am logged in as a random user
+      And I post the work "The course of true love never did run smooth" with fandom "October Daye Series - Seanan McGuire" with character "Tybalt (Toby Daye)" with second character "October Daye" with relationship "Toby Daye/Tybalt"
+     When the tag wrangler "lain" with password "lainnial" is wrangler of "October Daye Series - Seanan McGuire"
+       And I go to the tagwranglings page
+       And I follow "Relationships by fandom (1)"
+       And I check the wrangling option for "Toby Daye/Tybalt"
+       And I fill in "fandom_string" with "October Daye Series - Seanan McGuire"
+       And I press "Wrangle"
+     Then I should see "The following names are not canonical fandoms: October Daye Series - Seanan McGuire."
+
+  Scenario: more wrangling....Relationships canonically
+    Given the following typed tags exists
+        | name                                   | type         | canonical |
+        | Tybalt (Toby Daye)                     | Character    | true      |
+        | October 'Toby' Daye                    | Character    | true      |
+        | Toby Daye/Tybalt                       | Relationship | true      |
+        | October Daye Series - Seanan McGuire   | Fandom       | true      |
+      And a media exists with name: "Books & Literature", canonical: true
+      And I am logged in as a random user
+      And I post the work "The course of true love never did run smooth" with fandom "October Daye Series - Seanan McGuire" with character "Tybalt (Toby Daye)" with second character "October 'Toby' Daye" with relationship "Toby Daye/Tybalt"
+     When the tag wrangler "lain" with password "lainnial" is wrangler of "October Daye Series - Seanan McGuire"
+       And I go to the tagwranglings page
+       And I follow "Relationships by fandom (1)"
+       And I check the wrangling option for "Toby Daye/Tybalt"
+       And I fill in "fandom_string" with "October Daye Series - Seanan McGuire"
+       And I press "Wrangle"
+     Then I should see "The following tags were successfully wrangled to October Daye Series - Seanan McGuire: Toby Daye/Tybalt"
+
+  Scenario: more wrangling....Characters can be canonical
```

Comment:
Again, I think we need a better description for the scenario -- something like "Tag wranglers can mass wrangle characters into canonical fandoms from the Characters bin." (It occurs to me 0-use tags _might_ not show up in the wrangling bin, in which case I'd suggest shortening this by posting one work with two character tags and not creating the character tags in the `Given the following typed tags exists` step.)